src/averageAges.js

```diff
@@ -36,8 +41,23 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let female = withChildren
```

Comment:
A lot of actions
You can just find women with children or women & calc average)
Try to make better